llvm/lib/Target/M68k/GISel/M68kCallLowering.cpp

```diff
@@ -25,6 +25,11 @@
 
 using namespace llvm;
 
+struct M68kFormalArgHandler : public M68kIncomingValueHandler {
```

Comment:
This should be wrapped into an anonymous namespace `namespace { }`, so it doesn't get exported.

Reply:
Done.